app/src/androidTest/java/com/simplemobiletools/calculator/MoneyActivityTest.java

```diff
@@ -84,6 +85,23 @@ public void taxWithoutGeoLocTest(){
         checkResult("25.99");
     }
 
+    @Test
+    public void testTipDialogDisplayed() {
+        onView(withId(R.id.btn_tip)).perform(click());
```

Comment:
can't you use the press method instead?

Reply:
That's what I did originally but I couldn't get it to open. idk why